editor/utils/tracking.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-console */
```

Comment:
> I don't think this belongs in a new root folder. Maybe as part of utils

For me a root folder is a separate module just like the packages repo, and I don't think we should add a lot of things to the `utils` module, I prefer smaller separate modules (like the ongoing PR for wordCounter)